cmd/kube-apiserver/app/server.go

```diff
@@ -114,6 +116,8 @@ func NewAPIServer() *APIServer {
 		APIRate:                10.0,
 		APIBurst:               200,
 		APIPrefix:              "/api",
+		ExpAPIPrefix:           "/exp",
```

Comment:
Nit, but a big one: In general, we don't abbreviate in the API. Spell it out: /experimental.